subcommand/server-acl-init/command_test.go

```diff
@@ -129,7 +116,7 @@ func TestRun_Defaults(t *testing.T) {
 func TestRun_TokensPrimaryDC(t *testing.T) {
 	t.Parallel()
 
-	cases := map[string]struct {
+	cases := []struct {
 		TokenFlag          string
 		ResourcePrefixFlag string
```

Comment:
I think remove this `ResourcePrefixFlag` and use the global `resourcePrefix` variable internally in the test. Then use `SecretName: resourcePrefix + "-client-acl-token"` (for example) for the cases